recipes/google-cloud-aiplatform/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "google-cloud-aiplatform" %}
+{% set version = "1.10.0" %}
+{% set sha256 = "b6901122638f13f7c4ec4218a4d41855766f056d169bb29377492175f384c5ed" %}
+{% set dev_url = "https://github.com/googleapis/python-aiplatform" %}
+{% set doc_url = "https://cloud.google.com/python/docs/reference/aiplatform/latest" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - google-api-core-grpc >=1.26.0,<3.0.0dev
+    - proto-plus >=1.10.1
+    - packaging >=14.3
+    - google-cloud-storage >=1.32.0,<2.0.0dev
+    - google-cloud-bigquery >=1.15.0,<3.0.0dev
+
+test:
+  imports:
+    - google.api_core
+    - google.cloud.aiplatform
+    - google.cloud.aiplatform.hyperparameter_tuning
+    - google.cloud.aiplatform.initializer
+    - google.cloud.aiplatform.jobs
+    - google.cloud.aiplatform.models
+    - google.cloud.aiplatform.pipeline_jobs
+    - google.cloud.aiplatform.schema
+    - google.cloud.aiplatform.training_jobs
+    - google.cloud.aiplatform.version
+    - google.cloud.aiplatform_v1
+    - google.cloud.aiplatform_v1beta1
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: {{ dev_url }}
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: Vertex AI SDK for Python
+  description: |
+    Google Vertex AI is an integrated suite of machine learning tools and services for building and using ML models 
+    with AutoML or custom code. It offers both novices and experts the best workbench for the entire machine learning 
+    development lifecycle.
+  doc_url: {{ doc_url }}
+  dev_url: {{ dev_url }}
+
+extra:
+  recipe-maintainers:
+    - parthea
+    - morgandu
```

Comment:
@morgandu Please confirm that you'd like to be added as a maintainer (as per checklist)

Reply:
Thanks for the PR, please add me as maintainer.